packages/react-components/src/components/utils/touch.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+
+/**
+ * @private
+ */
+export const useLongTouch = (ref: React.RefObject<HTMLElement>, onLongTouch: () => void): void => {
```

Comment:
Can we resuse the existing useLongPress here isntead? https://github.com/Azure/communication-ui-library/blob/2a9fe90161c315216e1584ea15b8e66b7ce9c5a3/packages/react-components/src/components/utils/useLongPress.ts

Or can we update useLongPress to do anything this is also doing so we just have one implementation